### PR TITLE
Commit WIP of CoC implementation documentation

### PR DESCRIPTION
**Pull request by jorydotcom**
per the work we did in the May 5 meeting, here's a WIP PR of CoC implementation documentation for onboarding projects

### CODE REVIEW

File: Project-CoC-Implementation-Details.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# OpenJSF Project Code of Conduct Implementation Details
+
+This document is a guide for Projects to safely implement the OpenJS Foundation Code of Conduct policy.  
+
+## Adding the Foundation Code of Conduct to your Project
+
+Adopting the Foundation Code of Conduct is a requirement for all OpenJSF projects. Completing this action will meet the first requirement in the [Project Onboarding Checklist](https://github.com/openjs-foundation/project-onboarding/blob/master/ISSUE_TEMPLATE/00-project-onboarding-checklist-template.md).
+
+Projects may add the Foundation's Code of Conduct a few different ways - the choice is left the project based on what best suits its size and community.
```
Comment (joesepi) [inline Project-CoC-Implementation-Details.md]: left **to** the project ...

**Review by joesepi — APPROVED** [review]
LGTM. Thanks!

**Review by mcollina — COMMENTED** [review]
I would recommend using a .github folder to specify the CoC.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by joesepi** [comment]
I'm going to try to capture what I believe the gist of our conversation in yesterday's meeting was:
the most ideal situation would be to have the CoC be in an org .github directory that permeates to the repos in the github org. This is great from a provider perspective as it is automatically added to every new and existing project. From a consumer standpoint (end user, person who forks repo, someone looking to report a violation, etc) having a CODE_OF_CONDUCT.md in the projects root feels like the best and most obvious solution.

Ideally, these two scenarios being solved by one approach -- using the .github directory would also mirror a file in the repo root. This is not a reality today, but I've made the feature request to a friend at Github.

The alternative is to use the .github but also put a CODE_OF_CONDUCT.md file at root manually and add a link to the .github CoC in this file.

**Pull request merged**